support/helpers/api/getToken.ts

```diff
@@ -0,0 +1,88 @@
+import { request } from '@playwright/test'
+import config from '../../../playwright.config'
+
+const baseUrl = config.use.baseURL
+const clientId = 'web'
+
+const logonUrl = `${baseUrl}/signin/v1/identifier/_/logon`
+const tokenUrl = `${baseUrl}/konnect/v1/token`
+const redirectUrl = `${baseUrl}/oidc-callback.html`
+
+export async function getAdminToken(username: string, password: string): Promise<string> {
+  const context = await request.newContext()
+
+  const continueUrl = await logonUser(context, username, password)
+  const code = await getAuthorizationCode(context, continueUrl)
+  return await requestAccessToken(context, code)
+}
+
+async function logonUser(context: any, username: string, password: string): Promise<string> {
```

Comment:
Needs to be imported from `@playwright/test`.

```suggestion
async function logonUser(context: APIRequestContext, username: string, password: string): Promise<string> {
```